### DISCUSSION TITLE
Updating Wingbits

### DISCUSSION THREAD

**Discussion by jayscovill** [ketilmo/balena-ads-b#103] [General]
Hello, is there a supported way of updating the Wingbits feeder to the main branch they require now? I tried using the update script they provide in the container but it borked it.

**Comment by shawaj** [comment]
@jayscovill you just need to go to the page here - https://github.com/ketilmo/balena-ads-b#part-14--updating-to-the-latest-version

and click on deploy with balena. then choose the pre-existing fleet you want to deploy to and it will deploy the updated code

**Comment by shawaj** [comment]
also see https://github.com/ketilmo/balena-ads-b/issues/101 for ref

**Reply by jayscovill** [reply]
Oh wow, I don't know how I missed that post! Thanks, appreciate the help!

**Comment by jayscovill** [comment]
Thanks @shawaj. Just to confirm,  is that going to update the Wingbits container to the latest version? Or just the BalenaOS version?  I just built my balena-ads-b on a Pi 3 B+ last night so if that does update the Wingbits container I may already have the latest?

**Reply by shawaj** [reply]
clicking that button will update all the services of balena-ads-b to the latest. however, yes if you did it last night you probably have the latest already - as the update for wingbits was applied in #101 by @ketilmo a couple days ago

**Reply by jayscovill** [reply]
Thanks again, appreciate you taking the time to do my work for me lol.
